sdk/src/types/block/output/native_token.rs

```diff
@@ -251,6 +255,13 @@ impl NativeTokens {
     }
 }
 
+// TODO: should we also impl `WorkScore` for `NativeToken` for plain consistency?
```

Comment:
Don't think there's any point since the value apparently is constant

Reply:
Removed